### ISSUE TITLE
Training loss unstable when training in SDD dataset

### ISSUE THREAD

**Issue by mulplue** [meaten/FlowChain-ICCV2023#6]
Hi, when I'm trying to train model on SDD dataset, I found the training loss is unstable like this: 
![image](https://github.com/meaten/FlowChain-ICCV2023/assets/113297398/30016c84-d344-40e6-b5c0-1aee8cf20177)

By carefully checking, I found the problem is that there're some noise data in train_trajnet.pkl, for example:
![20240116-160107](https://github.com/meaten/FlowChain-ICCV2023/assets/113297398/66ecb5e5-5ba7-4c80-b646-05cd3e979810)

I filter these abnormal data and train again(follow the same setting), the training loss curve becomes smooth, but the ADE/FDE is 14.17/27.95 which is much more higher than the resutls of paper. 
I wonder if you've met this problem and how you fix this, thank you!

**Comment by meaten** [comment]
Hi @mulplue,

I never changed or filtered data for fair comparison with previous methods.
One possible cause of instability is the differences of hyperparameters.
I trained FlowChain with tuned hyperparameters saved in optuna.db. ( for example, learning rate 5e-3 -> 4e-4 by tuning )
If you trained the model with default hyperparameters, learning rate is very high, and may causes unstable training.
